hugr-llvm/src/extension/int.rs

```diff
@@ -95,7 +95,102 @@ fn emit_int_op<'c, H: HugrView>(
                 .build_int_neg(arg.into_int_value(), "")?
                 .as_basic_value_enum()])
         }),
+        IntOpDef::iabs => emit_custom_unary_op(context, args, |ctx, arg, _| {
```

Comment:
how bad is `ipow`?

Reply:
It's more annoying than this - `ipow` isn't implemented in llvm, so we have to turn it into a list of multiplications ourselves. I'm thinking about how to do this in hugr-llvm without too much code duplication - doing it as a lowering hugr pass is also an option...